lib/services/presets.go

```diff
@@ -202,6 +196,66 @@ func NewPresetAuditorRole() types.Role {
 	return role
 }
 
+// NewPresetReviewerRole returns a new pre-defined role for reviewer. The
+// reviewer will be able to review all access requests.
+func NewPresetReviewerRole() types.Role {
+	enterprise := modules.GetModules().BuildType() == modules.BuildEnterprise
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V6,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetReviewerRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Review access requests",
+		},
+		Spec: types.RoleSpecV6{
+			Options: types.RoleOptions{
+				CertificateFormat: constants.CertificateFormatStandard,
+				MaxSessionTTL:     types.NewDuration(apidefaults.MaxCertDuration),
+				PortForwarding:    types.NewBoolOption(true),
+				ForwardAgent:      types.NewBool(true),
+				BPF:               apidefaults.EnhancedEvents(),
+				RecordSession:     &types.RecordSession{Desktop: types.NewBoolOption(true)},
+			},
+			Allow: types.RoleConditions{
+				ReviewRequests: defaultAllowAccessReviewConditions(enterprise)[teleport.PresetReviewerRoleName],
+			},
+		},
+	}
+	role.SetLogins(types.Allow, []string{"no-login-" + uuid.New().String()})
+	return role
+}
+
+// NewPresetRequesterRole returns a new pre-defined role for requester. The
+// requester will be able to request all resources.
+func NewPresetRequesterRole() types.Role {
+	enterprise := modules.GetModules().BuildType() == modules.BuildEnterprise
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V6,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetRequesterRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Request all resources",
+		},
+		Spec: types.RoleSpecV6{
+			Options: types.RoleOptions{
+				CertificateFormat: constants.CertificateFormatStandard,
+				MaxSessionTTL:     types.NewDuration(apidefaults.MaxCertDuration),
+				PortForwarding:    types.NewBoolOption(true),
+				ForwardAgent:      types.NewBool(true),
+				BPF:               apidefaults.EnhancedEvents(),
```

Comment:
Why do we need these options on the requester/reviewer roles?

Reply:
Bad copy paste. :-X